src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -107,10 +114,24 @@
               ><img src="../assets/images/thirdparty/cavatica.png" /> Cavatica
             </a>
           </div>
-          <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'NFL'">
+
+          <div
+            fxFlex="100%"
+            *ngIf="config.FEATURES.enableLaunchWithGalaxy && workflow?.descriptorType === WorkflowModel.DescriptorTypeEnum.Gxformat2"
+          >
+            <app-multi-cloud-launch
+              [cloudInstances]="cloudInstances"
+              [usersCloudInstances]="usersCloudInstances"
+              [languagePartner]="partner.GALAXY"
+              [appendToLaunchUrl]="'%23workflow/' + workflow?.full_workflow_path"
```

Comment:
Doesn't the version need to be set too? I know it's not working on their side yet, but we should have it working correctly on our side.

Reply:
I didn't include the version because it's not working on their end, but I thought it'd be okay for us to not use it for now. The user can select the version once they are on the galaxy side of things.

Reply:
I feel like it's worth doing. It seems like it might not be trivial (you can't just append) and worth figuring out while you're here rather than a hotfix. Presumably DNAstack will have a similar pattern in any case?

Reply:
trs_server is set by putting it as the url field in the cloud instance endpoint. Mine is set to `https://usegalaxy.org/workflows/trs_import?trs_server=dockstore.org&trs_id=`

The Galaxy people gave this as an example of launch with working, although it doesn't https://workflowhub.eu/workflows/110, but it shows the version as just getting appended. I didn't append the version here because if you launch with that, a little error message pops up on the Galaxy side that the version doesn't exist, but you can just select the version after that. I'm not sure where they're at with this. I asked on Gitter today, but haven't received a response yet.

I thought we could have this enabled without the version/error message. But if we only want it enabled with the version working, then I can add it. Or if the small error message the user would see is fine, then I can add it anyway.

Reply:
I thought when I was playing with it there was no error, but yes, I see it, so best to not have it.